error-filter/README.md

```diff
@@ -0,0 +1,68 @@
+# Opossum Error-Filter Example
+
+This example shows how we can use the errorFilter option of opossum.
+
+### Server
+
+The server is an express.js application that exposes 3 endpoints.
+
+<table>
+<tbody>
+  <tr>
+    <td>/</td>
+    <td>returns a 200 status code.</td>
+  </tr>
+  <tr>
+    <td>/not-found</td>
+    <td>returns a 404 status code.</td>
+  </tr>
+  <tr>
+    <td>/error</td>
+    <td>returns a 500 status code.</td>
+  </tr>
+</tbody>
+</table>
+
+### Client
+
+The client consists of a simple node.js module making a request for each of the server's available endpoints and when all request complete, it logs the breaker's stats.
+
+### Steps
+
+```
+$ npm start
+
```

Comment:
I think a line here `open another terminal` and run npm run client :+1:

Reply:
@helio-frota Done!